armory/data/datasets.py

```diff
@@ -1365,13 +1387,21 @@ def ucf101(
     cache_dataset: bool = True,
     framework: str = "numpy",
     shuffle_files: bool = True,
+    max_frames: int = None,
```

Comment:
do we want the default to be all frames or set to 512? I think the former makes more sense, which is what this PR is currently doing, but just want to confirm since I might not remember the full context of this issue

Reply:
Default for dataset should be all; setting to 512 in scenario_configs, however.